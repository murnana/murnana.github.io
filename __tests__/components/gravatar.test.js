import React from "react"
import renderer from "react-test-renderer"
import Gravatar from "../../src/components/gravatar"

describe("Menu", () => {
  it("render", () => {
    const tree = renderer
      .create(<Gravatar hash={"00000000000000000000000000000000"} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
