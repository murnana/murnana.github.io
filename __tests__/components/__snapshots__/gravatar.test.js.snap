// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu render 1`] = `
<img
  alt="Gravatar"
  src="https://www.gravatar.com/avatar/00000000000000000000000000000000?s=undefined"
/>
`;
